DESCRIPTION

```diff
@@ -19,8 +19,8 @@ Authors@R: c(
     person("Dony", "Unardi", , "unardid@gene.com", role = "rev"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Utilities for data transformations compatible with 'teal'
-    applications.
+Description: Provides a `teal_data` class as a unified data model for `teal` applications
```

Comment:
I have seen somewhere that "`" is forbidden. Better to come back to the original quotes.

Reply:
None of the available packages has "`" whereas there are a lot of with "'".

(I just did ctrl+f on https://cran.r-project.org/web/packages/available_packages_by_name.html)

Reply:
Noted. Thanks for this. I am so used to backticks that I didn't even think twice about the DESCRIPTION file

This change was done in 75d07c203b2fc92b36e31ca8fe98d23b8e226714 and resubmitted 2 days ago